protostar/commands/test/cheatcodes/reflect_cheatcode.py

```diff
@@ -0,0 +1,224 @@
+from typing import Any, Callable, List, Dict, NamedTuple, Tuple, Type, Union
+from copy import deepcopy
+import dataclasses
+
+from starkware.cairo.lang.vm.vm_consts import is_simple_type, VmConstsReference
+from starkware.cairo.lang.vm.relocatable import RelocatableValue
+from starkware.cairo.lang.compiler.ast.cairo_types import (
+    CairoType,
+    TypeStruct,
+    TypePointer,
+    TypeFelt,
+)
+from starkware.cairo.lang.compiler.identifier_definition import (
+    StructDefinition,
+    MemberDefinition,
+)
+
+from protostar.starknet.cheatcode import Cheatcode
+
+
+@dataclasses.dataclass
+class _ReflectTreeNode:
```

Comment:
don't these Python std modules do a lot of stuff going on here?
- https://docs.python.org/3/library/pprint.html
- https://docs.python.org/3/library/textwrap.html

Reply:
Sadly, pretty print doesn't work for NamedTuples.